sentences/pt-br/_common.yaml

```
@@ -1,33 +1,45 @@
     no_device_class_in_area_exposed: "Desculpe, nenhuma {{ device_class }} na área {{ area }} está exposta"
-
-    no_floor: "Desculpe, não existe nenhum piso chamado {{ floor }}"
-    no_domain_in_floor: "Desculpe, não existe nenhum(a) {{ domain }} no piso {{ floor }}."
-    no_device_class_in_floor: "Desculpe, não existe nenhum(a) {{ device_class }} no piso {{ floor }}."
-    no_entity_in_floor: "Desculpe, não existe nenhum dispositivo chamado {{ entity }} no piso {{ floor }}."
-    no_entity_in_floor_exposed: "Desculpe, {{ entity }} no piso {{ floor }} não está exposto."
-    no_domain_in_floor_exposed: "Desculpe, nenhum(a) {{ domain }} no piso {{ floor }} está exposto."
     no_device_class_in_floor_exposed: "Desculpe, nenhum(a) {{ device_class }} no piso {{ floor }} está exposto."
-    duplicate_entities_in_floor: "Desculpe, existem vários dispositivos chamados {{ entity }} no piso {{ floor }}."
 
+    # Used when multiple (exposed) devices have the same name    
```

yamllint

[error] trailing spaces

(trailing-spaces)